cypress/support/api/requests/Checkout.js

```diff
@@ -21,6 +21,7 @@ export function createCheckout({
     billingAddress,
     "billingAddress"
   );
+  cy.log("w funkcji tworzenia");
```

Comment:
Remove it.

Reply:
checkout shouldn't be changed under this task, changes removed